websub-ballerina/http_service.bal

```diff
@@ -15,47 +15,30 @@
 // under the License.
 
 import ballerina/http;
-import ballerina/log;
-import ballerina/jballerina.java;
 
 # Represents an underlying HTTP Service.
-service class HttpService {
-    private SubscriberService subscriberService;
-    private string? secretKey;
-    private boolean isSubscriptionValidationDeniedAvailable = false;
-    private boolean isSubscriptionVerificationAvailable = false;
-    private boolean isEventNotificationAvailable = false;
+isolated service class HttpService {
+    private final HttpToWebsubAdaptor httpToWebSubAdaptor;
+    private final string? secretKey;
+    private final boolean isSubscriptionValidationDeniedAvailable;
+    private final boolean isSubscriptionVerificationAvailable;
+    private final boolean isEventNotificationAvailable;
 
     # Initializes `websub:HttpService` endpoint.
     # ```ballerina
-    # websub:HttpService httpServiceEp = check new ('service, "sercretKey1");
+    # websub:HttpService httpServiceEp = check new (handler, "sercretKey1");
     # ```
     # 
-    # + subscriberService - The current `websub:SubscriberService` instance
+    # + httpToWebSubAdaptor - The `websub:HttpToWebsubAdaptor` instance which used as a wrapper to execute service methods
     # + callback - Optional `secretKey` value to be used in the content distribution verification
     # + return - The `websub:HttpService` or an `error` if the initialization failed
-    isolated function init(SubscriberService subscriberService, string? secretKey) returns error? {
-        self.subscriberService = subscriberService;
+    isolated function init(HttpToWebsubAdaptor httpToWebSubAdaptor, string? secretKey) returns error? {
```

Comment:
These functions aren't exposed to the public right? We don't necessarily need API doc comments here.